CONTRIBUTING.md

```diff
@@ -1,40 +1,3 @@
-# Contributing to honeycomb-beeline
+# Contributing Guide
 
-Features, bug fixes and other changes to this project are gladly accepted.
-Please open issues or a pull request with your change. Remember to add your name
-to the CONTRIBUTORS file!
-
-All contributions will be released under the Apache License 2.0.
-
-## Developing
```

Comment:
Maybe move this deleted development environment and releasery info to DEVELOPING.md and RELEASING.md before we merge the conforming CONTRIBUTING.md?

Reply:
Oh wow, a repo with a health file! I think we should verify that release instructions are good next time we release before creating releasing.md. Updated contributing to keep prior art -- I imagine we will enhance contribution guides in other repos going forward.